scripts/gen_dockerfile_test.py

```diff
@@ -146,6 +155,9 @@ def test_get_app_config_invalid(app_yaml):
     # Python version
     (_BASE_APP_CONFIG._replace(dockerfile_python_version='_my_version'), True,
      'python_version=python_my_version'),
+    # Python version
```

Comment:
Can you change this comment to `# python-compat runtime`

Reply:
Done